### PR TITLE
A few minor Quality of Life changes to LibFX

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

**Comment by grimreaper** [comment]
I found this library through your blog; started poking around, and figured I'd submit a couple of changes.
Hope you find them useful.

File: src/main/java/org/codefx/libfx/control/webview/WebViewHyperlinkListener.java [hunk 1]
```diff
@@ -2,19 +2,20 @@
...
 
-import javafx.scene.web.WebEngine;
-import javafx.scene.web.WebView;
-
 import javax.swing.event.HyperlinkEvent;
 import javax.swing.event.HyperlinkListener;
 
+import javafx.scene.web.WebEngine;
+import javafx.scene.web.WebView;
+
```
Comment (nipafx) [inline src/main/java/org/codefx/libfx/control/webview/WebViewHyperlinkListener.java]: Can you return this to the original import order (not that it is better in any way, I just want to reduce the chance of "no-op changes").

**Comment by nipafx** [comment]
Great, thanks! :+1: Wow, you can tell that I wrote this code before being all down with Java 8.

(I didn't touch this project in a while, which is sad because I still have interesting stuff lying around in unmerged branches...)

File: src/main/java/org/codefx/libfx/control/webview/WebViewHyperlinkListener.java (hunk 1 above)
Comment (grimreaper) [reply src/main/java/org/codefx/libfx/control/webview/WebViewHyperlinkListener.java]: resolved

**Comment by nipafx** [comment]
Merged into `develop` in 0a35d07.

Thanks again and sorry for this taking so ridiculously long.

**Pull request closed without merging**